### PR TITLE
Add doc for running WOPI server locally

### PR DESCRIPTION
**Pull request by LovisaLugnegard**
Added config files and documentation for running the wopi server locally.

### CODE REVIEW

File: docker/etc/wopi/wopiserver.defaults.conf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#
+# wopiserver.conf - basic working configuration for a docker image
+#
+
+[general]
+storagetype = cs3
+port = 8880
+allowedclients = localhost glpdesk.cern.ch glpdesk 172.17.0.1
```
[the remark this answers is not in the document]
Comment (glpatcern) [reply docker/etc/wopi/wopiserver.defaults.conf]: Edit: this line is gone as we know

File: src/wopiserver.py [hunk 2]
```diff
@@ -78,7 +78,7 @@ def init(cls):
       cls.log.addHandler(loghandler)
       # read the configuration
       cls.config = configparser.ConfigParser()
-      with open('/etc/wopi/wopiserver.defaults.conf') as fdef:
+      with open('docker/etc/wopi/wopiserver.defaults.conf') as fdef:
```
[the remark this answers is not in the document]
Comment (LovisaLugnegard) [reply src/wopiserver.py]: This is already removed :)

[the remark this answers is not in the document]
Comment (glpatcern) [reply src/wopiserver.py]: yeah I can imagine - I meant to have posted the above 2 weeks ago, and got fooled by Github's changed workflow :-D

File: README.md [hunk 3]
```diff
@@ -47,3 +47,14 @@ By default, the local storage is tested. To run the tests, use the standard pyth
 2. Go to the test folder `cd test`
 3. Configure `wopiserver-test.conf` according to your Eos setup. The provided defaults are valid at CERN.
 4. Run the tests: `WOPI_STORAGE=xroot python3 test_storageiface.py`
+
+## Run the WOPI server locally
+1. Install all requirements listed in requirement.txt
+2. Add log file directory: `sudo mkdir /var/log/wopi/`
+3. Add the log file: `sudo vim /var/log/wopi/wopiserver.log`
+4. Make sure you have write permissions to the log file `sudo chmod a+rwx /var/log/wopi/wopiserver.log`
```
Comment (glpatcern) [inline README.md:55]: You can remove step 3. and change this to `sudo chmod a+rwx /var/log/wopi`

File: README.md [hunk 4]
```diff
@@ -47,3 +47,14 @@ By default, the local storage is tested. To run the tests, use the standard pyth
...
+5. Create the folder for the wop config `mkdir /etc/wopi/``
```
Comment (glpatcern) [inline README.md:56]: wop -> wopi

File: README.md [hunk 5]
```diff
@@ -47,3 +47,14 @@ By default, the local storage is tested. To run the tests, use the standard pyth
...
+6. Copy `iopsecret` and `wopiscret` from `etc` to `/etc/wopi/`
```
Comment (glpatcern) [inline README.md:57]: Rather, generate `iopsecret` and `wopisecret` with random strings

File: README.md [hunk 6]
```diff
@@ -47,3 +47,14 @@ By default, the local storage is tested. To run the tests, use the standard pyth
...
+7. Copy `etc/wopiserver.conf` to `/etc/wop/wopiserver.defaults.conf`
```
Comment (glpatcern) [inline README.md:58]: Copy `wopiserver.conf` to `/etc/wopi/...` (typo)

File: README.md [hunk 7]
```diff
@@ -47,3 +47,14 @@ By default, the local storage is tested. To run the tests, use the standard pyth
...
+8. Update the `wopiserver.defaults.config` file according to your needs.
```
Comment (glpatcern) [inline README.md:59]: Rather: Create a `wopiserver.conf` with the needed parameters. Start from `docker/etc/wopiserver.conf`, and make sure that at least an application provider URL is configured (e.g. `codeurl` for Collabora).

File: README.md (hunk 5 above)
Comment (LovisaLugnegard) [reply README.md]: Maybe a bit overkill for a local deployment? But ok.

**Comment by glpatcern** [comment]
Looks all good, I'd have kept a separate `wopiserver.conf` on top of the defaults one, but in the end this is for a simple local deployment, so it's more than enough. Merging.

**Pull request merged**